docs/06_how-to-guides/30_key-value-api/kv_table/index.md

```diff
@@ -10,20 +10,20 @@ A `datastore key value table on-chain`, or a `KV Table`, serves as a storage loc
 [[caution | Alpha version]]
 | `Key-Value Table` is designated as `alpha` and should not be used in production code.
 
-The `object definition` consists of a list of `properties` and each property is stored on the corresponding row in the table. The properties of the objects are also referred to as `fields`.
+The `object definition` consists of a list of `properties` and each data member is stored on the corresponding row in the table. The properties of the objects are also referred to as `fields`.
```

Comment:
why is properties accented?

Reply:
+ object definition ...?

Reply:
+ fields

Reply:
properties is wrongly used here; data members is the right term